src/cleanvision/utils/utils.py

```diff
@@ -16,25 +16,25 @@
 ]  # filetypes supported by PIL
 
 
+# todo: make recursive an option
 def get_filepaths(
     dir_path: str,
 ) -> List[str]:
-    """
-    Used in initialization of ImageDataset Class
-    Obtains image files of supported types and
-    sorts them based on filenames numerically and alphabetically
+    """Gets paths of all image files in the dir_path recursively.
+     All image files with extension in TYPES are allowed.
+     Returns a sorted list of sorted filepaths
 
 
     Parameters
     ----------
-    dir_path: str (an attribute of ImageDataset Class)
-    a string represening the current working directory
+    dir_path: str
+        Path to the dir containing image files, can be relative or absolute path
 
 
     Returns
     -------
-    sorted_names: list[str]
-    a list of image filenames sorted numerically and alphabetically
+    list of str
```

Comment:
```suggestion
    sorted_names: list[str]
```

Reply:
Is the particular reason why you replaced the old part of this with "list of str" ?

The old version looks more polished and would be preferred style for rest of documentation IMO

Reply:
I replaced this because I was editing while going through the numpy docstyle page and one of the examples was like this. But in other parts I stuck to other type of annotation and didn't come back to this.